CMakeLists.txt

```diff
@@ -161,6 +175,7 @@ ExternalProject_Add(psi4_core
               -DPYTHON_EXECUTABLE=${PYTHON_EXECUTABLE}
               -DPYTHON_INCLUDE_DIR=${PYTHON_INCLUDE_DIR}
               -DPYTHON_LIBRARY=${PYTHON_LIBRARY}
+              -DPYBIND11_ROOT=${PYBIND11_ROOT}
```

Comment:
That you're having to do all this FindPYBIND11 stuff is my fault for not getting the Config pushed upstream. I'll get on that.